test/unit/period/generators/bi-monthly_spec.js

```diff
@@ -35,27 +35,27 @@ describe('Bi-monthly period', () => {
                 startDate: '2017-03-01',
                 endDate: '2017-04-30',
                 name: `March - April 2017`,
-                id: '201703B',
+                id: '201702B',
```

Comment:
I knew this looked wrong :P But it's what the old one generated so i thought it would be correct. 

Something like.. March is the 3rd month and then Bi-Monthly. But i guess thats not how it works?